### PR TITLE
Throw exception when rad command is too far from current position

### PR DESCRIPTION
**Pull request by Ishadijcks**
Changed the value from 0.2 to 0.05

Our max speed is 1.7 rad/s, at 100hz this means we need to move 0.017 rad per instruction at most. 0.05 was chosen to have a bit of a buffer.

NOTE: This means that if the controller has an error of more than 0.05 rad it will also trigger this limit. This needs to be tested on the exoskeleton

### CODE REVIEW

File: march_hardware/src/IMotionCube.cpp [hunk 1]
```diff
@@ -97,10 +97,11 @@ void IMotionCube::writeInitialSettings(uint8 ecatCycleTime)
 
 void IMotionCube::actuateRad(float targetRad)
 {
-  if (std::abs(targetRad - this->getAngleRad()) > 0.2)
+  if (std::abs(targetRad - this->getAngleRad()) > 0.05)
```
Comment (JitskedeVries) [inline march_hardware/src/IMotionCube.cpp]: 0.05 is probably okay, should be tested on the exoskeleton

Comment (Ishadijcks) [reply march_hardware/src/IMotionCube.cpp]:
```suggestion
  if (std::abs(targetRad - this->getAngleRad()) > 0.075)
```

File: march_hardware/src/IMotionCube.cpp [hunk 2]
```diff
@@ -97,10 +97,11 @@ void IMotionCube::writeInitialSettings(uint8 ecatCycleTime)
...
   {
-    ROS_ERROR("Target %f exceeds max difference of 0.2 from current %f", targetRad, this->getAngleRad());
-    return;
+    ROS_ERROR("Target %f exceeds max difference of 0.05 from current %f for slave %d", targetRad, this->getAngleRad(),
```
Comment (Ishadijcks) [inline march_hardware/src/IMotionCube.cpp]:
```suggestion
    ROS_ERROR("Target %f exceeds max difference of 0.075 from current %f for slave %d", targetRad, this->getAngleRad(),
```

File: march_hardware/src/IMotionCube.cpp [hunk 3]
```diff
@@ -97,10 +97,11 @@ void IMotionCube::writeInitialSettings(uint8 ecatCycleTime)
...
+              this->slaveIndex);
+    throw std::runtime_error("Target exceeds max difference of 0.05 from current position");
```
Comment (Ishadijcks) [inline march_hardware/src/IMotionCube.cpp]:
```suggestion
    throw std::runtime_error("Target exceeds max difference of 0.075 from current position");
```

**Review by JitskedeVries — APPROVED** [review]

**Pull request merged**